storage/config.go

```diff
@@ -52,8 +52,15 @@ type Config struct {
 	// inputs is accelerated. The size of the cache is large so understand how to configure the cache.
 	// TODO provide some default config choices
 	// If this section is skipped, Caching is disabled
-	Cache  CachingConfig `json:"cache"`
-	Limits LimitsConfig  `json:"limits" pflag:",Sets limits for stores."`
+	Cache             CachingConfig     `json:"cache"`
+	Limits            LimitsConfig      `json:"limits" pflag:",Sets limits for stores."`
+	DefaultHTTPClient *HTTPClientConfig `json:"defaultHttpClient" pflag:",Sets the default http client config."`
```

Comment:
this makes it sound like a client, not a config for a client.

Reply:
Similar to Cache?